_posts/2021-04-28-#17331.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: pr
+date: 2021-04-28
+title: "Use effective values throughout coin selection"
+pr: 17331
+authors: [achow101]
+components: ["wallet"]
+host: Xekyo
+status: upcoming
+commit: 4ac1add
+---
+
+## Notes
+
+- Wallets maintain their balance as a pool of Unspent Transaction Outputs
+  (UTXOs).
+
+- _Coin selection_ refers to the process of picking UTXOs from the wallet's UTXO
+  pool to fund a transaction.
+
+- Beyond the primary goal of funding a transaction, the secondary goals of coin
+  selection are:
+
+    - Minimizing short term and long term fees:  
+      We want to both pay the least right now, but we also want to be good
+      stewards of our UTXO pool and not rack up large UTXO counts that will
+      translate to huge future transaction fees.
+
+    - Maintaining financial privacy:  
+      Ideally, our transactions leak as little information about ourselves and
```

Comment:
This is a bit abstract, maybe add a little about linking addresses, e.g. spending UTXOs together usually means it's from the same wallet.

Reply:
Rewrote the section